lib/units.js

```diff
@@ -81,7 +81,7 @@ const UNIT_SIZES = {
 };
 
 const sizeToBytes = (
-  // Convert string with units to int
+  // Convert string with file size to int
```

Comment:
This is not necessarily a file size.

Reply:
Also, s/int/integer, to be consistent with other places where you've changed it.